### PR TITLE
Add unit and + to stepper filter selection title

### PR DESCRIPTION
**Pull request by shredlocker**
# Why?

Stepper filter selection title is only `3`, we want it to be `3+ soverom`

# What?

- Added alternative unit to stepper configuration.
- Use the alternative unit to display current stepper filter selection.

# Show me

### Before

Screenshot 2019-03-26 at 14 09 44

### After

Screenshot 2019-03-26 at 14 07 54

### CODE REVIEW

File: Sources/FINNSetup/FilterMarkets/FilterMarketRealestate.swift [hunk 1]
```diff
@@ -270,7 +270,7 @@ extension FilterMarketRealestate: FINNFilterConfiguration {
 
         switch filterKey {
         case .noOfBedrooms:
-            return StepperFilterConfiguration(minimumValue: 0, maximumValue: 6, unit: "stk")
+            return StepperFilterConfiguration(minimumValue: 0, maximumValue: 6, unit: "stk", alternativeUnit: "soverom")
```
Comment (vadymmarkov) [inline Sources/FINNSetup/FilterMarkets/FilterMarketRealestate.swift]: Why don't we use "stk" for bedrooms?

Comment (shredlocker) [reply Sources/FINNSetup/FilterMarkets/FilterMarketRealestate.swift]: I asked on Trello, I hope they say we can use "stk".

Comment (shredlocker) [reply Sources/FINNSetup/FilterMarkets/FilterMarketRealestate.swift]: Now it only has the `+` at the end.

File: Sources/Charcoal/Selection/FilterSelectionStore.swift [hunk 2]
```diff
@@ -140,7 +140,8 @@ extension FilterSelectionStore {
                 return ["\(lowValue ?? "...") - \(highValue ?? "...")"]
             }
         case .stepper:
-            return value(for: filter).map({ [$0] }) ?? []
+            guard let lowValue: Int = value(for: filter) else { return [] }
+            return ["\(lowValue)+"]
```
Comment (vadymmarkov) [inline Sources/Charcoal/Selection/FilterSelectionStore.swift]: `return value(for: filter).map({ ["\($0)+"] }) ?? []` should also work, but keep `guard` if you like it more 😉

Comment (shredlocker) [reply Sources/Charcoal/Selection/FilterSelectionStore.swift]: I changed if `if let` instead to be more consistent with the other cases. That map thing is a nice one liner though 😄

**Pull request merged**